### DISCUSSION TITLE
Add an option during initial setup to choose between Launcher and Application Library as the default super key action

### DISCUSSION THREAD

**Discussion by Walrusz** [pop-os/pop#2068] [Ideas]
While I think that the Launcher is a very powerful tool, many users might find the Application Library to be more intuitive, so I propose an option during initial setup that lets users decide what the super key action should be without having to set it later in the settings.

**Comment by why-not-try-calmer** [comment]
Makes sense. Up voted!

**Comment by amifelipek** [comment]
I don't think this is good UX, new users won't know what the launcher nor the application library look like and behave so having an option to change that would be kinda pointless IMO.

**Reply by Walrusz** [reply]
I was thinking it could include screenshots for them, like how it already shows what the Launcher looks like and have short descriptions beneath them. But maybe you're right.
